cli/installsp/template.go

```diff
@@ -0,0 +1,127 @@
+package installsp
+
+// Template provides the base template for the `linkerd install-sp` command.
+const Template = `apiVersion: linkerd.io/v1alpha1
+kind: ServiceProfile
+metadata:
+  name: linkerd-controller-api.{{.Namespace}}.svc.cluster.local
+  namespace: {{.Namespace}}
```

Comment:
If the command is executed before the `linkerd install` command, the `linkerd` namespace (or whatever namespace was chosen) won't exist. Will k8s accept this .yaml in that case? Or will it error out and say that `namespace doesn't exist`?

Reply:
`linkerd install-sp` would be called *after* `linkerd install`. The service profiles are just nice to have (as they get the per-route metrics).